motioneye/sendmail.py

```diff
@@ -90,7 +90,7 @@ def on_media_files(media_files):
 
             # filter out non-recent media files
             media_files = [
-                m for m in media_files if abs(m['timestamp'] - timestamp) < timespan
+                m for m in media_files.result() if abs(m['timestamp'] - timestamp) < timespan
```

Comment:
Apply pre-commit black format:
```suggestion
                m
                for m in media_files.result()
                if abs(m['timestamp'] - timestamp) < timespan
```